isis/CMakeLists.txt

```diff
@@ -564,6 +564,11 @@ add_custom_target(export ALL COMMAND ${CMAKE_COMMAND} -E copy_if_different
   ${CMAKE_SOURCE_DIR}/appdata/export/* ${CMAKE_BINARY_DIR}/appdata/export)
 add_dependencies(isis export)
 
+# Add custom build target to copy modified Camera.plugin files to the build/lib/Camera.plugin file
```

Comment:
Question: This looks like a new solution, the Camera plugins were partially working before. Where was the old code?